tests/DbaaS/verifyDBaaS_test.js

```diff
@@ -344,3 +345,34 @@ Failed to register pmm-agent on PMM Server: Post "https://https:%2F%2F1.2.3.4/v1
   },
 );
 
+Scenario('@PMM-T1512 Verify tooltips work properly for DBaaS page @dbaas',
+  async ({
+    I, dbaasPage, adminPage, settingsAPI,
+  }) => {
+    let turnOffDbaas;
+
+    if (!(await settingsAPI.getSettings('dbaas_enabled'))) {
```

Comment:
This check can be removed, dbaas tests run with -e ENABLE_DBAAS=1 so situation where dbaas is disabled and needs to be enabled cannot happen.

Reply:
Removed. @beatahandzelova could you check again?